sympy/vector/coordsysrect.py

```diff
@@ -14,7 +15,7 @@ class CoordSysCartesian(Basic):
     """
```

Comment:
This class could be renamed `CoordSys3D`. For compatibility purposed, add a function with called `CoordSysCartesian` that later calls `CoordSys3D`